.travis.yml

```diff
@@ -77,6 +77,7 @@ env:
   - CLIENT=saucelabs:chrome COMMAND=test
   - CLIENT=saucelabs:safari:6 COMMAND=test
   - CLIENT="saucelabs:internet explorer:10:Windows 8" COMMAND=test
+  - CLIENT="saucelabs:MicrosoftEdge:20.10240:Windows 10" COMMAND=test
```

Comment:
Did you have problems with the version or did you pick that from the local build you tested? if we could leave off the version here and pick up latest stable it would be ideal